.gitignore

```diff
@@ -42,4 +42,11 @@ Thumbs.db
 
 .next
 .turbo
-node_modules
\ No newline at end of file
+node_modules
+
+**/.xatarc
+apps/**/xata.codegen.ts
+.env
+.env.*
+
```

Comment:
Why are we ignoring this?

Reply:
ignoring the every variation of `.env.` so users need to explicitly decide to commit things if they choose so

maybe I could allow-list `.env.template` and `.env.example` as they are common patterns?